cli/src/main/kotlin/OrtMain.kt

```diff
@@ -128,14 +138,14 @@ class OrtMain : CliktCommand(name = ORT_NAME, epilog = "* denotes required optio
         // Make the parameter globally available.
         printStackTrace = stacktrace
 
-        // Make the OrtConfiguration available to subcommands.
-        currentContext.findOrSetObject {
-            OrtConfiguration.load(
-                configArguments,
-                configFile
-            )
+        // Make options available to subcommands.
+        val globalOptionsForSubcommands = currentContext.findOrSetObject {
+            mutableMapOf<String, Any>()
         }
 
+        globalOptionsForSubcommands[OPTION_CONFIG] = OrtConfiguration.load(configArguments, configFile)
+        globalOptionsForSubcommands[OPTION_FORCE_OVERWRITE] = forceOverwrite
```

Comment:
I would prefer a type-safe approach instead of using a map, this would also lead to cleaner code. For example:
```
data class MainOptions(
  val config: OrtConfiguration,
  val forceOverwrite: Boolean
)
```